### PR TITLE
Remove redundancy in heading in bug report issue form 

### PR DESCRIPTION
**Pull request by opusforlife2**
### Pull Request Checklist

- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog

The issue form currently generates a bug report that contains the "What happened?" heading twice. See https://github.com/vector-im/element-android/issues/4047#issue-1001300890.

This PR edits the culprit heading and its subheadings to remove the redundancy and clarify what is meant.

### CODE REVIEW

**Comment by opusforlife2** [comment]
@bmarty @kittykat Is it okay now?

**Review by bmarty — APPROVED** [review]
OK for me

**Review by ouchadam — APPROVED** [review]

**Comment by ouchadam** [comment]
@bmarty should we also sync this with iOS? https://github.com/vector-im/element-ios/blob/develop/.github/ISSUE_TEMPLATE/bug.yml

**Review by kittykat — CHANGES_REQUESTED** [review]

**Comment by kittykat** [comment]
> @bmarty should we also sync this with iOS? https://github.com/vector-im/element-ios/blob/develop/.github/ISSUE_TEMPLATE/bug.yml

Yes, please submit the same changes for element-ios and element-web (which has two forms) after this one is finalised and add me as the reviewer for those

File: changelog.d/4076.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove redundant heading from bug report issue form
```
Comment (bmarty) [inline changelog.d/4076.misc:1]: This is not really corresponding to the (new) change.

**Review by bmarty — COMMENTED** [review]
@opusforlife2 will you update this PR? Else I'm afraid we will close it.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -23,12 +23,12 @@ body:
   - type: textarea
     id: result
     attributes:
-      label: What happened?
+      label: Intended vs. Actual Result
```
[the remark this answers is not in the document]
Comment (opusforlife2) [reply .github/ISSUE_TEMPLATE/bug.yml]: Done. Is it so that those who use online translators get better translations?

**Comment by opusforlife2** [comment]
> Yes, please submit the same changes for element-ios and element-web (which has two forms) after this one is finalised and add me as the reviewer for those

@kittykat Is this addressed to me?

> will you update this PR? Else I'm afraid we will close it.

@bmarty I'm here on weekends only. Since you guys have the weekend off, there is bound to be a mismatch between our work times, sadly.

**Comment by bmarty** [comment]
@opusforlife2 thanks for the update!

**Comment by bmarty** [comment]
> > Yes, please submit the same changes for element-ios and element-web (which has two forms) after this one is finalised and add me as the reviewer for those
> 
> @kittykat Is this addressed to me?

I think so, yes :)

> 
> > will you update this PR? Else I'm afraid we will close it.
> 
> @bmarty I'm here on weekends only. Since you guys have the weekend off, there is bound to be a mismatch between our work times, sadly.

Sure, sorry for my unnecessary pressure. We are trying to reduce the number of opened (and sometimes staled) PRs.

**Comment by bmarty** [comment]
I let @kittykat approve and merge the PR

**Review by kittykat — APPROVED** [review]

**Comment by kittykat** [comment]
@opusforlife2 I'm trying out slightly different wording in element-web, can you have a look at that and see what you think? Please submit a PR for element-ios and add me as a reviewer :)

**Pull request merged**